MAUI/Picker/populating-items.md

```diff
@@ -0,0 +1,218 @@
+    countryName.Add("United Kingdom");
+    countryName.Add("Australia");
+    countryName.Add("Germany");
+    countryName.Add("France");
+    countryName.Add("Japan");
+    countryName.Add("China");
+    countryName.Add("Brazil");
+    PickerColumn countryColumn = new PickerColumn()
+    {
+        HeaderText = "Select Country",
+        ItemsSource = countryName,
+        SelectedIndex = 1,
+    };
+    ObservableCollection<object> cityName = new ObservableCollection<object>();
+    cityName.Add("Chennai");
+    cityName.Add("Mumbai");
+    cityName.Add("Delhi");
+    cityName.Add("Kolkata");
+    cityName.Add("Bangalore");
+    cityName.Add("Hyderabad");
+    cityName.Add("Pune");
+    cityName.Add("Ahmedabad");
+    cityName.Add("Jaipur");
+    cityName.Add("Lucknow");
+    cityName.Add("Chandigarh");
+    PickerColumn cityColumn = new PickerColumn()
+    {
+        HeaderText = "Select City",
+        ItemsSource = cityName,
+        SelectedIndex = 1,
+    };
+    this.Picker.Columns.Add(countryColumn);
+    this.Picker.Columns.Add(cityColumn);
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set items text style customization
+
+The .NET MAUI Picker control, both selected and unselected items text color, font size, font family, font attributes can be customize as shown in the following code.
+
+### Selected item customization
+
+Selected item text style can be customized by setting `SfPicker.SelectedTextStyle` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+    <sfPicker:SfPicker>
+        <sfPicker:SfPicker.SelectedTextStyle>
+            <sfPicker:PickerTextStyle FontSize="16" FontAttributes="Bold" TextColor="Gray">
+            </sfPicker:PickerTextStyle>
+        </sfPicker:SfPicker.SelectedTextStyle>
+    </sfPicker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    this.Picker.SelectedTextStyle.FontSize = 16;
+    this.Picker.SelectedTextStyle.FontAttributes = FontAttributes.Bold;
+    this.Picker.SelectedTextStyle.TextColor = Colors.Gray;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Unselected item customization
+
+Unselected item text style can be customized by setting `SfPicker.TextStyle` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+    <sfPicker:SfPicker>
+        <sfPicker:SfPicker.TextStyle>
+            <sfPicker:PickerTextStyle FontSize="17" FontAttributes="Italic" TextColor="Green">
+            </sfPicker:PickerTextStyle>
+        </sfPicker:SfPicker.TextStyle>
+    </sfPicker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    this.Picker.TextStyle.FontSize = 17;
+    this.Picker.TextStyle.FontAttributes = FontAttributes.Italic;
+    this.Picker.TextStyle.TextColor = Colors.green;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Adding custom view to items
+
+In picker control, the items can be customized with custom view of all item by `ItemTemplate`, and assign custom view to the `ItemTemplate`.
```

Comment:
In picker control, the items can be customized with a custom view of all items by `ItemTemplate` and assign a custom view to the `ItemTemplate`.

Reply:
Content modified